format/negentropy/negentropy.go

```diff
@@ -0,0 +1,114 @@
+package negentropy
+
+// https://github.com/hoytech/negentropy
+
+import (
+	"embed"
+	"math"
+	"time"
+
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/interp"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+//go:embed negentropy.md
+var negFS embed.FS
+
+func init() {
+	interp.RegisterFormat(
+		format.Negentropy,
+		&decode.Format{
+			Description: "Negentropy message",
+			DecodeFn:    decodeNegentropyMessage,
+			Groups:      []*decode.Group{},
+		})
+	interp.RegisterFS(negFS)
+}
+
+const (
+	version         = 0x61
+	fingerprintSize = 16
+)
+
+var modeMapper = scalar.SintMapSymStr{
+	0: "skip",
+	1: "fingerprint",
+	2: "idlist",
+}
+
+type timestampTranslator struct{ last time.Time }
+
+func (tt *timestampTranslator) MapSint(s scalar.Sint) (scalar.Sint, error) {
+	if s.Actual == 0 {
+		s.Description = "infinity"
+		tt.last = time.Unix(math.MaxInt64, 0)
+		return s, nil
+	} else {
+		timestamp := tt.last.Add(time.Second * time.Duration(s.Actual-1))
+		s.Description = timestamp.UTC().Format(time.DateTime + " UTC")
```

Comment:
Most other decoder that format timestamps uses `.Format(time.RFC3339)`, would that work?